### PR TITLE
Fixes #28381 - Fixes typo related to pre_migrations tag

### PR DESCRIPTION
**Pull request by jameerpathan111**

### CODE REVIEW

File: definitions/procedures/foreman/fix_corrupted_roles.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module Procedures::Foreman
   class FixCorruptedRoles < ForemanMaintain::Procedure
     metadata do
       for_feature :foreman_database
-      tags :pre_migration
+      tags :pre_migrations
```
Comment (kgaikwad) [inline definitions/procedures/foreman/fix_corrupted_roles.rb]: As respective checks(`CheckCorruptedRoles` and `InvalidReportAssociations`) with tag  `pre-upgrade` are available, I don't think we will need to apply tag `pre_migrations` to these procedures (`FixCorruptedRoles` and `InvalidReportAssociations`).

Could you please remove these tags?

File: definitions/procedures/foreman_openscap/invalid_report_associations.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@ class InvalidReportAssociations < ForemanMaintain::Procedure
     metadata do
       param :ids_to_remove, 'Ids of reports to remove', :required => true
       for_feature :foreman_openscap
-      tags :pre_migration, :foreman_openscap, :openscap_report_associations
+      tags :pre_migrations, :foreman_openscap, :openscap_report_associations
```
Comment (kgaikwad) [inline definitions/procedures/foreman_openscap/invalid_report_associations.rb]: ditto

Comment (jameerpathan111) [reply definitions/procedures/foreman_openscap/invalid_report_associations.rb]: removed tag

File: definitions/procedures/foreman/fix_corrupted_roles.rb (hunk 1 above)
Comment (jameerpathan111) [reply definitions/procedures/foreman/fix_corrupted_roles.rb]: removed `pre_migration` tag.

**Comment by kgaikwad** [comment]
@jameerpathan111, looks good to me now. Could you please rebase this branch?

**Comment by jameerpathan111** [comment]
> @jameerpathan111, looks good to me now. Could you please rebase this branch?

Done.

**Comment by kgaikwad** [comment]
Thank you @jameerpathan111

**Pull request merged**